services/orders_create/lambda_function.py

```diff
@@ -37,13 +25,13 @@ def lambda_handler(event, context):
                 'total_cost': decimal.Decimal('0.0')
             }
         )
-        res = str(json.dumps(response, cls=DecimalEncoder))
+        res = str(json.dumps(response, default=str))
         statusCode = 200
         body = json.dumps({
             'order_id': order_id
         })
         print(f'put_item result: {res}')
-    except ClientError as e:
+    except Exception as e:
```

Comment:
Is there any particular reason why this is widened? To report all errors properly?

Reply:
indeed, catching only the ClientError here was not enough for handling all the potential errors that could be thrown inside the try clause, it's better to have it like this and let the lambda return the 400 code to the client rather than fail and perhaps return a different failure code (eg. 403 for key error that was not previously caught for example)